css-flexbox/css-flexbox-quiz-notes.md

```diff
@@ -0,0 +1,40 @@
+# css-flexbox-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What is the default `flex-direction` of a `flex` container?
+
+- What is the default `flex-wrap` of a `flex` container?
+
+- What axis do the `justify-content` properties work on?
+
+- What axis do the `align-items` properties work on?
+
```

Comment:
please don't forget to answer the quiz questions :)

Reply:
Okay, I just made the changes to include the answer to the quiz questions. Thanks!